### PR TITLE
Makes nest ghosting clearer

### PR DESCRIPTION
**Pull request by Erikos66**
## What Does This PR Do

Clarifies that the on-screen prompt will let you ghost, and adds a tgui alert with the option to ghost to boot.

Big thanks to DGamer and Burza in coding chat for the help.

## Why It's Good For The Game

Lowers the chances someone uses the ghost verb to ghost, forfeiting respawnability.

## Images of changes

image

image

image

image

## Testing

Tested on a local server

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  
  
  <hr>

## Changelog

:cl:
add: new tgui alert for ghosting when nested.
tweak: changes text informing the player that they can ghost using the on-screen button/prompt.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm [hunk 1]
```diff
@@ -79,7 +79,10 @@
 			if(user && !isalien(user))
 				return
 			buckled_mob.throw_alert("ghost_nest", /atom/movable/screen/alert/ghost)
-			to_chat(buckled_mob, "<span class='ghostalert'>You may now ghost, you keep respawnability in this state. You will be alerted when you're removed from the nest.</span>")
+			to_chat(buckled_mob, "<span class='ghostalert'>You may now ghost by clicking on the ghost prompt on your screen. You will be alerted when you're removed from the nest.</span>")
```
Comment (Contrabang) [inline code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm:82]:
```suggestion
			to_chat(buckled_mob, "<span class='ghostalert'>You may now click on the ghost prompt on your screen to leave your body. You will be alerted when you're removed from the nest.</span>")
```

**Review by Contrabang — CHANGES_REQUESTED** [review]
I think this will make it even clearer

File: code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm (hunk 1 above)
Comment (Erikos66) [reply code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm:82]: Very agreed.

File: code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm [hunk 2]
```diff
@@ -79,7 +79,10 @@
 			if(user && !isalien(user))
 				return
 			buckled_mob.throw_alert("ghost_nest", /atom/movable/screen/alert/ghost)
-			to_chat(buckled_mob, "<span class='ghostalert'>You may now ghost, you keep respawnability in this state. You will be alerted when you're removed from the nest.</span>")
+			to_chat(buckled_mob, "<span class='ghostalert'>You may now click on the ghost prompt on your screen to leave your body. You will be alerted when you're removed from the nest.</span>")
+			if(tgui_alert (buckled_mob, "You may now ghost and keep respawnability, you will be notified if you leave the nest, would you like to do so?", "Ghosting", list("Yes", "No")) != "Yes")
```
Comment (lewcc) [inline code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm:83]:
```suggestion
			if(tgui_alert(buckled_mob, "You may now ghost and keep respawnability, you will be notified if you leave the nest, would you like to do so?", "Ghosting", list("Yes", "No")) != "Yes")
```

**Review by lewcc — APPROVED** [review]

**Review by Burzah — APPROVED** [review]

**Review by Erikos66 — COMMENTED** [review]
No idea how to resolve these request changes xD

**Review by Contrabang — APPROVED** [review]

**Pull request merged**